pex/pip.py

```diff
@@ -46,23 +47,37 @@ def execute_pip_isolated(args, cache=None, interpreter=None):
 
 
 def _calculate_package_index_options(indexes=None, find_links=None):
+  trusted_hosts = []
+
+  def maybe_trust_insecure_host(url):
+    url_info = urlparse.urlparse(url)
+    if 'http' == url_info.scheme:
+      # Implicitly trust explicitly asked for http indexes and find_links repos instead of requiring
+      # seperate trust configuration.
+      trusted_hosts.append(url_info.netloc)
```

Comment:
Do you think we should log when this happens? I'm not sure what `-v` level that would be, but maybe something people should know about and encourage them to see if there's an `https` version.

Reply:
In order for this to happen, they said something like `pex --index=http://my.index ...`. If they really mean to use the insecure index - say its internal - they probably would appreciate not being nagged. I think we'd need to support some way of saying "no, really, trust me" in that case.